app/views/data-returns/choose-your-file.html

```diff
@@ -75,11 +75,10 @@ <h1 class="heading-large"><span id="title">{{$title}}{{/title}}</span></h1>
         <p id="invalid-upload-msg" {{^hasInvalidUploads}}style="display: none;" {{/hasInvalidUploads}}>
         You must correct any errors before your files can be sent to the Environment Agency.
         </p>
-        <a id="continueBtn" class="button" accesskey="C"
-           {{#canContinue}}href="/file/confirm" {{/canContinue}}
-           {{^canContinue}}disabled="disabled" {{/canContinue}}
-        >Continue</a>
-        </p>
+
+        <form action="/file/confirm" method="get">
+            <input class="button" type="submit" value="Continue" id="continueBtn" {{^canContinue}}disabled="disabled" {{/canContinue}} />
```

Comment:
Need to add accesskey="C" to the input to maintain the kb-only navigation functionality